package.json

```diff
@@ -13,11 +13,11 @@
     "esri-leaflet": "^3.0.2",
     "leaflet": "^1.7.1",
     "leaflet-mouse-position": "^1.2.0",
-    "plotly.js": "^2.3.0",
+    "plotly.js": "^2.3.1",
```

Comment:
was there a reason for these package updates?

Reply:
no whoops sorry, i think i just accidentally typed npm update instead of npm install one time

Reply:
i can undo it

Reply:
If you haven't already you can leave it, but I'd like to avoid upgrading in the future unless we need to for a specific reason. Just to avoid potentially breaking something.

the '^' means that packages will update on `npm update` to any minor version changes, 2.1.9 to 2.2.0 for example. (so no major version upgrades which would be 2.2.0 to 3.0.0 for example), so it's a little safeguard. I'll go in a switch them to `~` which means the version would update on `npm update` to the most recent patch version (the very last number), but not for minor or major version changes. So those are usually just bug fixes which we'd probably want.